runners/gearpump/pom.xml

```diff
@@ -69,6 +69,7 @@
                   <dependenciesToScan>
                     <dependency>org.apache.beam:beam-sdks-java-core</dependency>
                   </dependenciesToScan>
+                  <argLine>-noverify</argLine>
```

Comment:
Is there a JIRA filed for this? We should sort it out at some point, and maybe someone with some expertise will get interested.